examples/kv-server/src/main/java/com/yahoo/gondola/demo/DemoService.java

```diff
@@ -125,7 +120,7 @@ public int getAppliedIndex() {
     }
 
     public void clearState() {
-        entries.clear();
+        logger.info("In demo application, no need to clear the internal storage.");
```

Comment:
In demo app, there is no persistent storage, the in-memory state is the source of truth. 
There is no need to clear the internal storage when leader changes.